cypress/e2e/models/migration/archetypes/archetype.ts

```diff
@@ -281,4 +281,23 @@ export class Archetype {
 
         return duplicatedArchetype;
     }
+
+    getAssociatedAppsCount() {
+        Archetype.open();
+        return cy
+            .get(tdTag)
+            .contains(this.name)
+            .closest(trTag)
+            .find('[data-label="Applications"] p')
+            .invoke("text")
+            .then((text) => {
+                const numberMatch = text.match(/\d+/);
+                if (numberMatch) {
+                    const appCount = parseInt(numberMatch[0], 10);
+                    cy.wrap(appCount).as("appCount");
+                } else {
+                    cy.wrap(null).as("appCount");
+                }
```

Comment:
This can be simplified:

```
const number = parseInt(numberMatch[0], 10);
cy.wrap(isNaN(number) ? null : number).as("appCount");
```

Reply:
Yes, ternary operator can be used instead of if/else

Reply:
Done